app/views/govuk_publishing_components/components/docs/cookie_banner.yml

```diff
@@ -17,34 +17,34 @@ examples:
     data: {}
   with_custom_content:
     data:
-      title: "Can we store analytics cookies on your device?"
-      text: "This is some custom text in my cookie banner which lets users know what we're using cookies for. I can also include a link to the <a class='govuk-link' href='/cookies'>cookies page</a>"
-      confirmation_message: "You’ve accepted all cookies."
+      title: Can we store analytics cookies on your device?
+      text: This is some custom text in my cookie banner which lets users know what we're using cookies for. I can also include a link to the <a class='govuk-link' href='/cookies'>cookies page</a>
+      confirmation_message: You’ve accepted all cookies.
 
   with_multi_paragraph_custom_content:
     data:
-      title: "Can we store analytics cookies on your device?"
+      title: Can we store analytics cookies on your device?
       text: ["This is some custom text in my cookie banner.","There are three paragraphs.","They are passed as an array"]
-      confirmation_message: "You’ve accepted all cookies."
+      confirmation_message: You’ve accepted all cookies.
 
   with_custom_cookie_preferences_href:
     data:
-      cookie_preferences_href: "/cookies"
+      cookie_preferences_href: /cookies
   in_services_asking_for_analytics_only:
     description: Use this type of banner in your service if it only uses cookies for analytics.
     data:
       title: Can we store analytics cookies on your device?
       text: Analytics cookies help us understand how our website is being used.
       confirmation_message: You’ve accepted all cookies. You can `<a class='govuk-link' href='/cookies'>change your cookie settings</a>` at any time.
       services_cookies:
-        "yes":
-          text: "Yes"
+        yes:
+          text: Yes
```

Comment:
I think that `yes` and `no` are being converted to booleans, thanks to YAMLs "Norway Problem"; this is what comes through in the guide:

![image](https://user-images.githubusercontent.com/1732331/178238561-1f770d96-88e1-4472-87ae-03e346f4676e.png)

The fix is to quote any instance of "yes" or "no":

```suggestion
        "yes":
          text: "Yes"
```